content/en/docs/release-oversight/disruption-testing.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Disruption Testing"
+description: An overview for how disruption tests work and are configured.
+---
```

Comment:
Since we moved the contents of this file to `disruption-testing/code-implementation.md` We should delete this file, or move any missing information to the `disruption-testing/code-implementation.md` file